technical_analysis/volatility.py

```diff
@@ -30,9 +30,12 @@ def bbands(l_args, s_ticker, s_interval, df_stock):
             print(f"The following args couldn't be interpreted: {l_unknown_args}\n")
             return
 
+        # create new figure
+        plt.figure()
```

Comment:
Although it seems like "duplicated code" I would put this just before the first plt.plot() on both cases

Reply:
I can go ahead and do this, but I'm not totally sure why, since placing the call before the conditional has the same effect as placing it on both sides. Wouldn't it be cleaner to just have it in one place?

Reply:
I won't spam you by hitting resolve on all the comments but I implemented this request for all the files you commented on. I'll go ahead and resolve only this one.